### ISSUE TITLE
org.checkerframework.org.plumelib.util.UniqueId contains invalid annotations in Checker Framework release

### ISSUE THREAD

**Issue by cushon** [typetools/checker-framework#3956]
The copy of `org.checkerframework.org.plumelib.util.UniqueId` in `./checker-framework-3.7.1/checker/dist/checker.jar` incorrectly contains two identical annotations on the receiver of `getClassAndUid`.

The source of that method looks fine: https://github.com/plume-lib/plume-util/blob/d5431badfd9acb389277262d21a8db99fa9f91ef/src/main/java/org/plumelib/util/UniqueId.java#L51

I suspect the problem is being introduced when that source is compile with the checker framework enabled, when it inserts inferred/defaulted annotations into the bytecode.

## Repro

`javap` shows two identical copies of `@UnknownInitialization`:

```
$ javap -v -p -cp ./checker-framework-3.7.1/checker/dist/checker.jar org.checkerframework.org.plumelib.util.UniqueId
...
  public default java.lang.String getClassAndUid();
...
    RuntimeVisibleTypeAnnotations:
      0: #27(#19=c#28): METHOD_RECEIVER
        org.checkerframework.checker.initialization.qual.UnknownInitialization(
          value=class Lorg/checkerframework/org/plumelib/util/UniqueId;
        )
...
      28: #27(#19=c#28): METHOD_RECEIVER
        org.checkerframework.checker.initialization.qual.UnknownInitialization(
          value=class Lorg/checkerframework/org/plumelib/util/UniqueId;
        )
```

Attempting to load the annotations on the receiver of `getClassAndUid` using core reflection results in the runtime correctly reporting that the annotations are invalid:

```java
public class T {
  public static void main(String[] args) throws Exception {
    System.err.println(
        Class.forName("org.checkerframework.org.plumelib.util.UniqueId")
            .getMethod("getClassAndUid")
            .getAnnotatedReceiverType());
  }
}
```

```
$ javac T.java 
$ java -cp .:./checker-framework-3.7.1/checker/dist/checker.jar T
Exception in thread "main" java.lang.annotation.AnnotationFormatError: Duplicate annotation for class: interface org.checkerframework.checker.initialization.qual.UnknownInitialization: @org.checkerframework.checker.initialization.qual.UnknownInitialization(value=org.checkerframework.org.plumelib.util.UniqueId.class)
        at java.base/sun.reflect.annotation.TypeAnnotationParser.mapTypeAnnotations(TypeAnnotationParser.java:380)
        at java.base/sun.reflect.annotation.AnnotatedTypeFactory$AnnotatedTypeBaseImpl.<init>(AnnotatedTypeFactory.java:140)
        at java.base/sun.reflect.annotation.AnnotatedTypeFactory.buildAnnotatedType(AnnotatedTypeFactory.java:64)
        at java.base/sun.reflect.annotation.TypeAnnotationParser.buildAnnotatedType(TypeAnnotationParser.java:78)
        at java.base/java.lang.reflect.Executable.getAnnotatedReceiverType(Executable.java:676)
        at T.main(T.java:6)
```
